test/MostUsed/MostUsedSpec.hs

```diff
@@ -5,94 +5,82 @@ module MostUsed.MostUsedSpec
 
 import Data.List (intercalate)
 import MostUsed
+import MostUsed.Parser.Bash as Bash
 import MostUsed.Parser.Zsh as Zsh
 import Test.Hspec
-
-commandNumber :: String
-commandNumber = "  452  "
-
-unparseableCommand :: String
-unparseableCommand = cmd "foo(){\\n # -X = \"whatever\", blah\\n"
-
-cmd :: String -> String
-cmd s = commandNumber ++ s
-
-cmd2 :: String -> String
-cmd2 s = "    1  " ++ s
+import Text.Megaparsec.String
 
 main :: IO ()
 main = hspec spec
 
-escapedNewline :: String
-escapedNewline = "\\n"
-
 spec :: Spec
-spec = do
+spec = describe "top level" $ do
+    commonSpec "Bash parser" Bash.items bashCommand
```

Comment:
It's pretty cool that I can run the same tests (`commonSpec`) with different parsers.